File: BRaVe-Mobile-Backend/Models/DistributionAssistance.cs
using System.Text.Json.Serialization;

namespace BRaVe_Mobile_Backend.Models
{
    public class DistributionAssistance
    {
        [JsonPropertyName("distributionId")]
        public int distributionId { get; set; }

        [JsonPropertyName("householdId")]
        public string householdId { get; set; }

        [JsonPropertyName("individualId")]
        public int individualId { get; set; }

        [JsonPropertyName("data")]
        public string assistances { get; set; }
    }
}

File: BRaVeMobile/app/src/main/java/intl/iom/bravemobile/models/activities/DistributionAssistance.java
package intl.iom.bravemobile.models.activities;

public class DistributionAssistance {

    public int distributionId;
    public String householdId;
    public int individualId;
    public String data;

    public DistributionAssistance(int distributionId, String householdId, int individualId, String data) {
        this.distributionId = distributionId;
        this.householdId = householdId;
        this.individualId = individualId;
        this.data = data;
    }

    //query = "CREATE TABLE IF NOT EXISTS tbl_distribution_assistances(
    // activity_code TEXT, id INT, household_id TEXT,
    // individual_id INT, data TEXT, primary key(activity_code, id, household_id, individual_id));";
    //        db.execSQL(query);
}
